app/models.py

```diff
@@ -1078,7 +1078,18 @@ def website_send_to(self):
         """
 
         # Prefer using contact name if possible
+        user = self.user
         name = self.name
+        email = self.website_email
+
+        if (
+            not user
+            or not SenderFormatEnum.has_value(user.sender_format)
+            or user.sender_format == SenderFormatEnum.AT.value
+        ):
+            email = email.replace("@", " at ")
+        elif user.sender_format == SenderFormatEnum.A.value:
```

Comment:
should we take into account `SenderFormatEnum.VIA` and `SenderFormatEnum.FULL` as well?

Reply:
I already thought about those two: `FULL` is already supported as it doesn't modify the email. `VIA` also uses the `@` but the text "via SimpleLogin" doesn't make sense for me here. What do you think?